contrib/lib/tests/static_files.rs

```diff
@@ -22,6 +22,8 @@ mod static_tests {
             .mount("/both", StaticFiles::new(&root, Options::DotFiles | Options::Index))
             .mount("/redir", StaticFiles::new(&root, Options::NormalizeDirs))
             .mount("/redir_index", StaticFiles::new(&root, Options::NormalizeDirs | Options::Index))
+            .mount("/no_extension_no_default", StaticFiles::new(&root, Options::None))
+            .mount("/no_extension_with_default", StaticFiles::new(&root, Options::None).default_type(ContentType::Plain))
```

Comment:
Do we need `Options` at all here? Why not just `StaticFiles::from(&root).with_default(ContentType::Plain)`?